Makefile

```diff
@@ -1,7 +1,7 @@
 CC=gcc
-CFLAGS=--std=c99 -g -O2 -Wall --pedantic `freetype-config --cflags` `sdl-config --cflags`
-LDFLAGS=`icu-config --ldflags`
-LIBS=-lcairo -lharfbuzz -lharfbuzz-icu `freetype-config --libs` `sdl-config --libs`
+CFLAGS=--std=c99 -g -O2 -Wall --pedantic `pkg-config --cflags` `sdl2-config --cflags`
```

Comment:
I think this should be `pkg-config --cflags freetype2` if I'm not mistaken

Reply:
Yup that's true. I had redundant call for `--cflags`